lib/matplotlib/tests/test_colors.py

```diff
@@ -591,6 +595,15 @@ def test_PowerNorm_translation_invariance():
     assert_array_almost_equal(pnorm(a - 2), expected)
 
 
+def test_powernorm_cbar_limits():
+    fig, ax = plt.subplots()
+    vmin, vmax = 300, 1000
+    data = np.arange(100*100).reshape(100, 100) + vmin
```

Comment:
Do we need 10000 points? That seems a bit large, though I didn't check how slow the test was.